src/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. While this is not a functional issue, it's generally good practice to maintain a consistent import order for better readability and maintainability. The original order seems to follow the pattern of importing external libraries first, followed by internal modules. It would be better to stick to that.
```diff
- import PropTypes from 'prop-types';
- import * as React from 'react';
+ import * as React from 'react';
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->